app/models/topic.rb

```diff
@@ -1,2 +1,3 @@
 class Topic < ApplicationRecord
+    validates :name, presence: true
```

Comment:
don't you want to validate the uniqueness of name to? I think you will need, since topics should be unique by name (in my opinion), what do you think?